docs/iris/example_code/Meteorology/inset_plot.py

```diff
@@ -0,0 +1,66 @@
+"""
+Test Data Showing Inset Plots
+=============================
+
+This example demonstrates the use of a single 3D data cube with time, latitude
+and longitude dimensions to plot a temperature series for a single latitude
+coordinate, with an inset plot of the data region.
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import iris
+import cartopy.crs as ccrs
+import iris.quickplot as qplt
+import iris.plot as iplt
+
+
+def main():
+    # Load the data
+    with iris.FUTURE.context(netcdf_promote = True):
+        cube1 = iris.load_cube(iris.sample_data_path('ostia_monthly.nc'))
+    # Slice into cube to retrieve data for the inset map showing the data region
```

Comment:
This line is too long for the style checker to be comfortable with, just wrap it onto a second comment line at around 78 characters should sort it.